src/test/mocks/events.ts

```diff
@@ -0,0 +1,37 @@
+import { CONTRACT_ADDRESSES } from '../../utils/constants.ts'
+
+// DataSetCreated(uint256,uint256)
+const DATA_SET_CREATED_TOPIC0 = '0x11369440e1b7135015c16acb9bc14b55b0f4b23b02010c363d34aec2e5b96281'
+
+// DataSetCreated(uint256,uint256,uint256,uint256,uint256,address,address,address,string[],string[])
+// TODO(update-abi) const DATA_SET_CREATED_TOPIC0 = '0xc90cb3863281dc6e2e16e74064ed2e0ab91144ccfe5c3492b8c33f58fe90d0db'
```

Comment:
I decided that updating this ABI was out of scope for this PR but I left the work I had done for this here and below to help out whoever does it.